cluster/manifests/wiz/005-connector-job.yaml

```diff
@@ -10,20 +10,21 @@ metadata:
     helm.sh/chart: wiz-broker-2.1.0
     application: "wiz"
     component: "connector"
+    deployment: "wiz-connector-agent"
```

Comment:
Actually, this should be `job: wiz-connector-agent` as we use the kind to avoid that job and deployment resources with the same name could overlap.

Sorry if I suggested `deployment`, then I didn't realize this was a Job manifest.